Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/MetadataPanel.java

```diff
@@ -0,0 +1,197 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.production.forms.dataeditor;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+
+import javax.faces.model.SelectItem;
+
+import org.kitodo.api.Metadata;
+import org.kitodo.api.dataeditor.rulesetmanagement.StructuralElementViewInterface;
+import org.kitodo.api.dataformat.IncludedStructuralElement;
+import org.kitodo.api.dataformat.MediaUnit;
+import org.kitodo.production.helper.Helper;
+
+/**
+ * Backing bean for the meta-data panel of the meta-data editor.
+ */
+public class MetadataPanel implements Serializable {
+
+    private String addMetadataKeySelectedItem = "";
+
+    private String addMetadataValue = "";
+
+    private Collection<Metadata> clipboard = new ArrayList<>();
+
+    private final RulesetSetupInterface rulesetSetup;
+
+    private FieldedMetadataTableRow logicalMetadataTable = FieldedMetadataTableRow.EMPTY;
+    private FieldedMetadataTableRow physicalMetadataTable = FieldedMetadataTableRow.EMPTY;
```

Comment:
Yes, both metadata tables are displayed concurrently. 

If you start the metadata editor with the code in this branch, you will see there are two tables in the metadata column in the middle, in two PrimeFaces `p:accordionPanel` instances. 

This makes sense because in a case where both structure trees (physical and logical) are displayed at the same time (with `separateMedia=true`), only the physical tree will include pages that can be selected. 
At the same time, the logical structure element containing the selected page will be selected in the logical structure tree. In this situation both metadata tables are required and show different metadata concurrently (one shows the metadata of the logical structure element, one of the selected media unit).

Reply:
Okay, now I understand that. Then forget all my comments on this.